resources/inputmaps/xbox-controller.json

```diff
@@ -1,14 +1,48 @@
 {
   "name": "Xbox Controller",
-  "idmatcher": "Xbox.*",
```

Comment:
I think we want to catch both Xbox and XInput here because I developed this map on OSX with a xbox controller attached so my guess is that both are used on different platforms.